src/screens/authentication/components/Auth/Title/styles.js

```diff
@@ -3,9 +3,15 @@ import { verticalScale } from 'react-native-size-matters'
 
 const styles = StyleSheet.create({
   container: {
-    position: 'absolute',
-    top: verticalScale(116)
+    marginTop: verticalScale(24),
+    marginBottom: verticalScale(50)
+  },
+  emailSent: {
+    marginTop: verticalScale(24),
+    marginBottom: verticalScale(31)
   }
 })
 
 export default styles
+
+//  ME quede en ver como se hace margin condicional porque no entra de las dos formas
```

Comment:
The solution you provided is good. Please remove this comment.

In general we do not use comments, as code should be crafted in such a way that comments are not necessary to explain it, and they also get outdated, so they end up explaining something that it is no longer reflected in code.

We only add comments to explain `why` something is done in a certain way. For example check the `bugFixes` files, one of them explains that there is a bug regarding timers in React Native, so we end up modifying the global timers, that needs a `why` comment.

Reply:
I wrote that so i could remerber where i left it the next day. Forgot to take it out haha